adcm_aio_client/_session.py

```diff
@@ -73,6 +87,16 @@ async def __aexit__(
         exc_value: BaseException | None = None,
         traceback: TracebackType | None = None,
     ) -> None:
+        """
+        Closes ADCM session. Parameters are identical to httpx __aexit__.
```

Comment:
What's the connection to `httpx` in here?

Description of regular arguments for exit/aexit looks redundant